src/SearchInput/SearchInput.tsx

```diff
@@ -58,18 +78,31 @@ export const SearchInput = forwardRef<HTMLInputElement, SearchInputProps>(
       initialState: null,
       stateProp: value,
     })
-
     const [searchQuery, setSearchQuery] = useState<string | null>(null)
 
+    const fuse = useMemo(() => {
```

Comment:
Question: have you considered adding the search function as a prop so that it is treeshakeable like @david-gomes5 suggested?

Reply:
I did, but after the discussion with design during the reviews this will most likely be rolled out as a default and move to more of an "opt out" implementation, so I'm not overly concerned with it being tree shakeable